tests/BaseRequestTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace RonasIT\Support\Tests;
+
+use ReflectionClass;
+use RonasIT\Support\BaseRequest;
+use RonasIT\Support\Tests\Support\Mock\TestModel;
+use RonasIT\Support\Tests\Support\Traits\TableTestStateMockTrait;
+
+class BaseRequestTest extends HelpersTestCase
+{
+    use TableTestStateMockTrait;
+
+    public function testGetOrderableFields()
+    {
+        $baseRequest = new BaseRequest();
+        $reflectionClass = new ReflectionClass($baseRequest);
+
+        $method = $reflectionClass->getMethod('getOrderableFields');
+        $orderableFields = $method->invoke($baseRequest, TestModel::class);
+
+        $modelFields = implode(',', TestModel::getFields());
+
+        $this->assertEquals($orderableFields, $modelFields);
```

Comment:
please assert method result without any additional logic inside the test

```suggestion
        $this->assertEquals('field1,field2,field3', $result);
```